aws/v11.5.1/release.diff

```diff
@@ -0,0 +1,63 @@
+  - name: chart-operator                                             - name: chart-operator
+    version: 0.12.4                                                    version: 0.12.4
+  - componentVersion: 1.16.5                                         - componentVersion: 1.16.5
+    name: cluster-autoscaler                                           name: cluster-autoscaler
+    version: 1.16.0                                                    version: 1.16.0
+  - componentVersion: 1.6.5                                          - componentVersion: 1.6.5
+    name: coredns                                                      name: coredns
+    version: 1.2.0                                                     version: 1.2.0
+  - componentVersion: 0.7.2                                          - componentVersion: 0.7.2
+    name: external-dns                                                 name: external-dns
+    version: 1.2.2                                                     version: 1.2.2
+  - componentVersion: 3.5.0                                          - componentVersion: 3.5.0
+    name: kiam                                                         name: kiam
+    version: 1.3.0                                                     version: 1.3.0
+  - componentVersion: 1.9.5                                          - componentVersion: 1.9.5
+    name: kube-state-metrics                                           name: kube-state-metrics
+    version: 1.1.0                                                     version: 1.1.0
+  - componentVersion: 0.3.3                                          - componentVersion: 0.3.3
+    name: metrics-server                                               name: metrics-server
+    version: 1.1.0                                                     version: 1.1.0
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.9.0                                                     version: 1.9.0
+  - componentVersion: 0.18.1                                         - componentVersion: 0.18.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.2.0                                                     version: 1.2.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.6.0                                                     version: 1.6.0
+  - name: aws-operator                                               - name: aws-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 8.7.3                                              <      version: 8.7.5
```

Comment:
There are 2 versions but only one change. How does that work? 🙈

Reply:
You mean the changelog doesn't show the changes from 8.7.4? I didn't get a chance to implement merging changelogs for intermediate releases yet, but I would like to do it soon.

Reply:
In this case I think we just skipped one version but I was wondering why and if we are not missing something in the changelogs. The diff is fine for me.

Reply:
dependency for aws-operator (k8scloudconfig) with the correct calc for max pods was added in 8.7.5 😞 and not 8.7.4

Reply:
Still my question is not answered, what happened in the version before?

Reply:
The aws-operator changelog shows:

- 8.7.5 changed: "Adjust number of host network pods on worker node for aws-cni"
- 8.7.4 fixed: "Adjust MAX_PODS for master and worker nodes to max IP's per ENI when using aws-cni"

I consider the two item as closely related and in my release notes curation work decided to summarize it all as

> Adjust `MAX_PODS` for master and worker nodes to max IPs per Elastic Network Interface (ENI)

If we lose important information here, please let me know.